### ISSUE TITLE
[Bug] Broken keyboard tab navigation

### ISSUE THREAD

**Issue by SirVeggie** [lobehub/sd-webui-lobe-theme#204]
### 💻 系统环境 | Operating System

Windows

### 🌐 浏览器 | Browser

Other

### 🐛 问题描述 | Bug Description

Brave browser.

Tab navigation does not work in the prompt box preventing keyboard navigation to the following settings like the negative prompt box. This causes unnecessary mouse use.

### 🚦 期望结果 | Expected Behavior

Pressing tab moves the focus to the negative prompt box.

### 📷 复现步骤 | Recurrence Steps

1) Focus positive prompt box
2) Press Tab
= Broken navigation

**Comment by SirVeggie** [comment]
It seems possible to press esc -> tab -> tab to get to the negative prompt box, but it is quite cumbersome.
